<?xml version="1.0" encoding="utf-8" standalone="no"?>
<!--This file represents the results of running a test suite-->
<test-results name="D:\Sample Projects\Web Apps\TestServiceSln\TestServiceSln\NUnitTestApplication\bin\Debug\NUnitTestApplication.dll" total="7" errors="0" failures="1" not-run="0" inconclusive="0" ignored="0" skipped="0" invalid="0" date="2013-06-13" time="14:40:30">
  <environment nunit-version="2.6.2.12296" clr-version="2.0.50727.5466" os-version="Microsoft Windows NT 6.1.7601 Service Pack 1" platform="Win32NT" cwd="C:\Program Files (x86)\NUnit 2.6.2\bin" machine-name="TAHYS-0135363-L" user="U0135363" user-domain="TLR" />
  <culture-info current-culture="en-US" current-uiculture="en-US" />
  <test-suite type="Assembly" name="D:\Sample Projects\Web Apps\TestServiceSln\TestServiceSln\NUnitTestApplication\bin\Debug\NUnitTestApplication.dll" executed="True" result="Failure" success="False" time="1.983" asserts="0">
    <results>
      <test-suite type="Namespace" name="NUnitTestApplication" executed="True" result="Failure" success="False" time="1.982" asserts="0">
        <results>
          <test-suite type="TestFixture" name="SampleTestClass" executed="True" result="Success" success="True" time="0.003" asserts="0">
            <results>
              <test-case name="NUnitTestApplication.SampleTestClass.SampleTestMethod" executed="True" result="Success" success="True" time="0.000" asserts="1" />
            </results>
          </test-suite>
          <test-suite type="TestFixture" name="TestApplication" executed="True" result="Failure" success="False" time="0.039" asserts="0">
            <results>
              <test-case name="NUnitTestApplication.TestApplication.TestListMethod" executed="True" result="Success" success="True" time="0.001" asserts="1" />
              <test-case name="NUnitTestApplication.TestApplication.TestMethod" executed="True" result="Success" success="True" time="0.000" asserts="1" />
              <test-case name="NUnitTestApplication.TestApplication.TestPersonListMethod" executed="True" result="Failure" success="False" time="0.001" asserts="1">
                <failure>
                  <message><![CDATA[  Expected: collection containing < <NUnitTestApplication.Person> >
  But was:  < <NUnitTestApplication.Person>, <NUnitTestApplication.Person> >
]]></message>
                  <stack-trace><![CDATA[at NUnitTestApplication.TestApplication.TestPersonListMethod() in D:\Sample Projects\Web Apps\TestServiceSln\TestServiceSln\NUnitTestApplication\TestApplication.cs:line 44
]]></stack-trace>
                </failure>
              </test-case>
              <test-case name="NUnitTestApplication.TestApplication.TestPersonMethod" executed="True" result="Success" success="True" time="0.001" asserts="0" />
            </results>
          </test-suite>
          <test-suite type="TestFixture" name="TestServiceApplication" executed="True" result="Success" success="True" time="1.862" asserts="0">
            <results>
              <test-case name="NUnitTestApplication.TestServiceApplication.TestLocalServiceMethod" executed="True" result="Success" success="True" time="1.859" asserts="0" />
            </results>
          </test-suite>
          <test-suite type="TestFixture" name="TestServices" executed="True" result="Success" success="True" time="0.057" asserts="0">
            <results>
              <test-case name="NUnitTestApplication.TestServices.TestServerServiceMethod" executed="True" result="Success" success="True" time="0.056" asserts="0" />
            </results>
          </test-suite>
        </results>
      </test-suite>
    </results>
  </test-suite>
</test-results>